### PR TITLE
feat: additional file types for uploading models

### PR DESCRIPTION
**Pull request by officialsahyaboutorabi**
## Pull Request Checklist

- [ ] **Description:** Briefly describe the changes in this pull request.
- [ ] **Changelog:** Ensure a changelog entry following the format of [Keep a Changelog](https://keepachangelog.com/) is added at the bottom of the PR description.
- [ ] **Documentation:** Have you updated relevant documentation?
- [ ] **Dependencies:** Are there any new dependencies? Have you updated the dependency versions in the documentation?

---

## Description

[Insert a brief description of the changes made in this pull request]

---

### Added

- [List any new features or additions]

### Fixed

- [List any fixes or corrections]

### Changed

- [List any changes or updates]

### Removed

- [List any removed features or files]

### CODE REVIEW

**Comment by justinh-rahb** [comment]
Did you test this? Ollama only supports GGUF.

File: src/lib/components/chat/Settings/Models.svelte [hunk 1]
```diff
@@ -589,7 +589,7 @@
 												on:change={() => {
 													console.log(modelInputFile);
 												}}
-												accept=".gguf"
+												accept=".gguf",".safetensors"
```
Comment (tjbck) [inline src/lib/components/chat/Settings/Models.svelte:592]: Shouldn't this be `.bin` and not `.safetensors`?

Comment (officialsahyaboutorabi) [reply src/lib/components/chat/Settings/Models.svelte:592]: Both of them might work. Although don't rely on me as I am not too sure. I will have a look into it.

Comment (tjbck) [reply src/lib/components/chat/Settings/Models.svelte:592]: If you could check I'd greatly appreciate it!

Comment (officialsahyaboutorabi) [reply src/lib/components/chat/Settings/Models.svelte:592]: Alright, let me know if there is anything else you need me to look at.

**Comment by tjbck** [comment]
I'll merge this to our dev branch but I'm pretty sure we can't just directly upload `.safetensors` files, feel free to try it out with dev branch docker image and let us know your results.

**Comment by officialsahyaboutorabi** [comment]
> I'll merge this to our dev branch but I'm pretty sure we can't just directly upload `.safetensors` files, feel free to try it out with dev branch docker image and let us know your results.

How can I set up the dev branch docker image?

**Pull request merged**